profile.md

```diff
@@ -234,7 +234,14 @@ The following properties are defined for usage within a Kubernetes deployment:
   Version: 2.14
 
   The unique identifier for the Kubernetes cluster from which the request
-  was sent. This property MUST be a non-empty string serialized as follows:
+  was sent. This property can be useful when a Service Broker tries to
+  call back to a Platform that is managing multiple Kubernetes clusters.
+  Typically, this value will be static for the lifetime of the cluster,
```

Comment:
There's no way to guarantee this from an implementation standpoint - I think merging this to the spec was a mistake now.